requests-for-comments/2022-11-09-nail-distribution.md

```diff
@@ -0,0 +1,71 @@
+
+It's rational to go deep on a smaller selection of distribution channels and trying to do them really well, rather than blasting out content onto every platform imaginable in a shallow way. We've found this quality > quantity approach works well for us when it comes to writing the content itself. 
+
+Another problem is that it's not clear whose responsibility on the team content distribution is, so it ends up being a bit ad hoc. This is easy to solve though. 
+
+## Context
+
+These are the places where we distribute content right now, and who does it:
+
+- PostHog LinkedIn - _Andy mainly, sometimes Joe or James H_
+- PostHog Twitter - _Andy mainly, not sure if anyone else does?_
+- James Hawkins' personal Twitter - _James H_
+- HogMail - _Andy_
+- User Slack - _random, usually the content author_
+- Hacker News - _random, usually the content author_
+- SEO (not really a 'distribution' channel per se, but included for completeness) - _Andy_
+
+- LinkedIn paid ads - _Charles, via Hey_
+- Twitter paid ads - _Andy, via Hey_
+- Various newsletter sponsorships - _Charles_
+
+These are the places we want to try distributing content in Q4:
+
+- TikTok - _James H, via Flowlync_
+- YouTube - _Ian_
+
+These are the places we have also considered and decided not to try for now:
+
+- Reddit
+- Other Slacks
+- Other Discords
+- Hacker Noon, Dev To, IndieHackers
+
+It's _a lot._ And it's not obvious which ones we really care about doing well at. 
+
+## Proposal
+
+I'm going to split this into 'let's get good at this channel' vs. 'we should post here out of habit, but we're not trying to win here'. I think we should try a different approach to ownership of these channels though (oo, controversial):
+
+### Let's get (or stay) good at this in Q4
+
+- SEO - _Andy_
+- HogMail - _Andy_
+- James' personal Twitter - _James H_
+- TikTok - _James H_
+- YouTube - _Ian_
+- Charles' personal LinkedIn (maybe) - _Charles_
+
+### Let's keep posting here out of habit/hygiene factor
+
+For _these_ channels, I think it should just be the responsibility of the person who created the content itself to post it in the above channels (or not - it's your call). That way, we're staying focused on the stuff we want to get good at.
+
+- PostHog LinkedIn
+- PostHog Twitter
+- User Slack
+- Hacker News
+- LinkedIn paid ads
+- Twitter paid ads
+
+## How will we measure success?
+
+I didn't get this far yet as I'd like feedback on the above first! But instinctively, it makes sense for each channel owner to just decide this themselves, rather than me being prescriptive about it. 
+
+## But Charles we are basically already doing this
+
+Ok, well I didn't see it written down anywhere and got confused, so it's worth ensuring this stuff is documented properly. But I bet we could do this better!
```

Comment:
One area I think we can definitely improve is in having a workable cadence for content distro. 

For example, knowing that every piece (or piece of a certain type) will be shared to channels X, Y and Z, and that we will aim to create a high-performing piece of A and B (e.g. Hacker News, YouTube) once a week/month/day. This doesn't have to be hard and fast, but could form part of how we judge success and set metrics.

Reply:
I'm hesitant to try to rigidify cadence. I feel like we have a good cadence that could only slow down by adding more structure to it. 

Instead of cadence, just having really clear responsibilities for channels (such as Andy SEO, Ian Tutorials, James Twitter) and goals for signups/traffic would create a natural cadence decided by person responsible for the channel.

Reply:
I'm with Ian on this one. At our current team size, I don't think it would help much and would mostly serve to slow me down.

Reply:
Yeah I think I agree - I think clarifying ownership is important, but prescribing a content schedule (at this stage at least) doesn't feel necessary.